src/main/java/com/jcabi/github/RtHooks.java

```diff
@@ -118,20 +119,27 @@ public Hook get(final int number) {
         return new RtHook(this.entry, this.owner, number);
     }
 
+    // @checkstyle ParameterNumberCheck (2 lines)
     @Override
     public Hook create(
         final String name,
         final Map<String, String> config,
+        final Iterable<Event> events,
         final boolean active
     ) throws IOException {
-        final JsonObjectBuilder builder = Json.createObjectBuilder();
+        final JsonObjectBuilder configs = Json.createObjectBuilder();
         for (final Map.Entry<String, String> entr : config.entrySet()) {
-            builder.add(entr.getKey(), entr.getValue());
+            configs.add(entr.getKey(), entr.getValue());
+        }
+        final JsonArrayBuilder evnts = Json.createArrayBuilder();
+        for (final Event event : events) {
```

Comment:
@llorllale We could rewrite this `for` using `events.forEach()`, WDYT?

Reply:
@paulodamaso I would be happy to use Java 8 features, but somewhere in the build (maybe parents?) the source level is being set to 1.6.